src/relayer/RelayerConfig.ts

```diff
@@ -108,28 +108,30 @@ export class RelayerConfig extends CommonConfig {
           );
         }
       });
-      Object.keys(this.inventoryConfig.tokenConfig).forEach((l1Token) => {
-        Object.keys(this.inventoryConfig.tokenConfig[l1Token]).forEach((chainId) => {
-          const { targetPct, thresholdPct, unwrapWethThreshold, unwrapWethTarget } =
-            this.inventoryConfig.tokenConfig[l1Token][chainId];
-          assert(
-            targetPct !== undefined && thresholdPct !== undefined,
-            `Bad config. Must specify targetPct, thresholdPct for ${l1Token} on ${chainId}`
-          );
-          assert(
-            toBN(thresholdPct).lte(toBN(targetPct)),
-            `Bad config. thresholdPct<=targetPct for ${l1Token} on ${chainId}`
-          );
-          this.inventoryConfig.tokenConfig[l1Token][chainId].targetPct = toBNWei(targetPct).div(100);
-          this.inventoryConfig.tokenConfig[l1Token][chainId].thresholdPct = toBNWei(thresholdPct).div(100);
-          if (unwrapWethThreshold !== undefined) {
-            this.inventoryConfig.tokenConfig[l1Token][chainId].unwrapWethThreshold = toBNWei(unwrapWethThreshold);
-          }
-          this.inventoryConfig.tokenConfig[l1Token][chainId].unwrapWethTarget = unwrapWethTarget
-            ? toBNWei(unwrapWethTarget)
-            : toBNWei(2);
+      if (this.inventoryConfig.tokenConfig !== undefined) {
+        Object.keys(this.inventoryConfig.tokenConfig).forEach((l1Token) => {
```

Comment:
This high level of nesting is pretty hard to read; you could maybe get around it with the following:
```suggestion
        Object.keys(this.inventoryConfig?.tokenConfig ?? {}).forEach((l1Token) => {
```

Reply:
done